plugins/net_plugin/net_plugin.cpp

```diff
@@ -4192,10 +4248,18 @@ namespace eosio {
 
                std::string extra_listening_log_info =
                      ", max clients is " + std::to_string(my->connections.get_max_client_count());
-                     
+
+               auto listen_addr = address;
+               auto limit = string("0");
+               if( std::count(address.begin(), address.end(), ':') > 1 ) {
+                  auto last_colon_location = address.rfind(':');
+                  listen_addr = std::string(address, 0, last_colon_location);
+                  limit = std::string(address, last_colon_location+1);
+               }
+
                fc::create_listener<tcp>(
-                     my->thread_pool.get_executor(), logger, accept_timeout, address, extra_listening_log_info,
-                     [my = my, addr = p2p_addr](tcp::socket&& socket) { my->create_session(std::move(socket), addr); });
+                     my->thread_pool.get_executor(), logger, accept_timeout, listen_addr, extra_listening_log_info,
+                     [my = my, addr = p2p_addr, limit = limit](tcp::socket&& socket) { fc_dlog( logger, "start listening on ${addr} with peer sync throttle ${limit}", ("addr", addr)("limit", limit)); my->create_session(std::move(socket), addr, limit); });
```

Comment:
Since you are only limiting incoming connections, you need to make sure any incoming duplicate connections are also unlimited.

Reply:
Won't incoming duplicates be closed?

Reply:
One side is picked to close. So it can be a configured connection but seen as incoming.

Reply:
Added throttle exemption for configured p2p-peer-addresses.